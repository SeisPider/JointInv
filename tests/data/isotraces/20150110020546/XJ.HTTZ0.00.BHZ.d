SURF96 R U X -1 75.00000 3.63778 0.20387
SURF96 R U X -1 70.00000 3.71596 0.19854
SURF96 R U X -1 65.00000 3.74651 0.18741
SURF96 R U X -1 60.00000 3.80323 0.17827
SURF96 R U X -1 55.00000 3.81021 0.16401
SURF96 R U X -1 50.00000 3.71941 0.14208
SURF96 R U X -1 48.00000 3.68389 0.13380
SURF96 R U X -1 46.00000 3.67158 0.12737
SURF96 R U X -1 44.00000 3.67366 0.12197
SURF96 R U X -1 42.00000 3.66585 0.11593
SURF96 R U X -1 40.00000 3.64112 0.10893
SURF96 R U X -1 36.00000 3.57680 0.09460
SURF96 R U X -1 34.00000 3.52351 0.08671
SURF96 R U X -1 32.00000 3.48498 0.07983
SURF96 R U X -1 30.00000 3.47682 0.07449
SURF96 R U X -1 27.00000 3.15290 0.05513
