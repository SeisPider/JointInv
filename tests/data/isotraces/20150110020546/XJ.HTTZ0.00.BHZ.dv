MFT96 R U T -1 75.852    3.7471    1.0000    4868.4     0.1658E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 74.473    3.7476    1.0000    4868.4     0.1910E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 73.143    3.7480    1.0000    4868.4     0.2175E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 71.860    3.7482    1.0000    4868.4     0.2450E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 70.621    3.7484    1.0000    4868.4     0.2734E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 69.424    3.7484    1.0000    4868.4     0.3025E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 68.267    3.7483    1.0000    4868.4     0.3318E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 67.148    3.7482    1.0000    4868.4     0.3613E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 66.065    3.7479    1.0000    4868.4     0.3907E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 65.016    3.7475    1.0000    4868.4     0.4198E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 64.000    3.7470    1.0000    4868.4     0.4484E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 63.015    3.7464    1.0000    4868.4     0.4765E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 62.061    3.7457    1.0000    4868.4     0.5040E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 61.134    3.7448    1.0000    4868.4     0.5308E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 60.235    3.7439    1.0000    4868.4     0.5570E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 59.362    3.7429    1.0000    4868.4     0.5827E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 58.514    3.7417    1.0000    4868.4     0.6081E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 57.690    3.7405    1.0000    4868.4     0.6334E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 56.889    3.7391    1.0000    4868.4     0.6588E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 56.110    3.7376    1.0000    4868.4     0.6846E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 55.351    3.7361    1.0000    4868.4     0.7112E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 54.613    3.7344    1.0000    4868.4     0.7389E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 53.895    3.7326    1.0000    4868.4     0.7679E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 53.195    3.7307    1.0000    4868.4     0.7986E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 52.513    3.7287    1.0000    4868.4     0.8314E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 51.848    3.7266    1.0000    4868.4     0.8665E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 51.200    3.7243    1.0000    4868.4     0.9041E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 50.568    3.7220    1.0000    4868.4     0.9446E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 49.951    3.7196    1.0000    4868.4     0.9880E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 49.349    3.7170    1.0000    4868.4     0.1035E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 48.762    3.7144    1.0000    4868.4     0.1084E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 48.188    3.7116    1.0000    4868.4     0.1138E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 47.628    3.7088    1.0000    4868.4     0.1194E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 47.080    3.7058    1.0000    4868.4     0.1254E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 46.545    3.7028    1.0000    4868.4     0.1316E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 46.022    3.6996    1.0000    4868.4     0.1382E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 45.511    3.6963    1.0000    4868.4     0.1449E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 45.011    3.6929    1.0000    4868.4     0.1519E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 44.522    3.6895    1.0000    4868.4     0.1590E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 44.043    3.6859    1.0000    4868.4     0.1661E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 43.574    3.6822    1.0000    4868.4     0.1733E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 43.116    3.6784    1.0000    4868.4     0.1805E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 42.667    3.6745    1.0000    4868.4     0.1875E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 42.227    3.6705    1.0000    4868.4     0.1943E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 41.796    3.6665    1.0000    4868.4     0.2008E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 41.374    3.6623    1.0000    4868.4     0.2070E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.960    3.6580    1.0000    4868.4     0.2129E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.554    3.6536    1.0000    4868.4     0.2182E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.157    3.6491    1.0000    4868.4     0.2231E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 39.767    3.6445    1.0000    4868.4     0.2275E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 39.385    3.6399    1.0000    4868.4     0.2314E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 39.010    3.6351    1.0000    4868.4     0.2347E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 38.642    3.6302    1.0000    4868.4     0.2375E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 38.280    3.6253    1.0000    4868.4     0.2397E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 37.926    3.6202    1.0000    4868.4     0.2415E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 37.578    3.6151    1.0000    4868.4     0.2427E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 37.236    3.6099    1.0000    4868.4     0.2436E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.901    3.6045    1.0000    4868.4     0.2442E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.571    3.5991    1.0000    4868.4     0.2445E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.248    3.5936    1.0000    4868.4     0.2445E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.930    3.5880    1.0000    4868.4     0.2445E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.617    3.5823    1.0000    4868.4     0.2445E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.310    3.5766    1.0000    4868.4     0.2445E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.009    3.5707    1.0000    4868.4     0.2447E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.712    3.5648    1.0000    4868.4     0.2452E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.420    3.5587    1.0000    4868.4     0.2460E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.133    3.5526    1.0000    4868.4     0.2471E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.851    3.5464    1.0000    4868.4     0.2486E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.574    3.5401    1.0000    4868.4     0.2506E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.301    3.5338    1.0000    4868.4     0.2529E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.032    3.5273    1.0000    4868.4     0.2556E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.768    3.5208    1.0000    4868.4     0.2585E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.508    3.5142    1.0000    4868.4     0.2616E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.252    3.5075    1.0000    4868.4     0.2647E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.000    3.5007    1.0000    4868.4     0.2678E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.752    3.4939    1.0000    4868.4     0.2707E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.508    3.4870    1.0000    4868.4     0.2732E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.267    3.4800    1.0000    4868.4     0.2753E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.030    3.4729    1.0000    4868.4     0.2769E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.797    3.4658    1.0000    4868.4     0.2778E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.567    3.4586    1.0000    4868.4     0.2779E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.341    3.4513    1.0000    4868.4     0.2772E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.118    3.4440    1.0000    4868.4     0.2757E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.898    3.4366    1.0000    4868.4     0.2733E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.681    3.4291    1.0000    4868.4     0.2700E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.468    3.4215    1.0000    4868.4     0.2659E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.257    3.4139    1.0000    4868.4     0.2609E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.050    3.4062    1.0000    4868.4     0.2552E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.845    3.3985    1.0000    4868.4     0.2487E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.643    3.3907    1.0000    4868.4     0.2417E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.444    3.3828    1.0000    4868.4     0.2340E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.248    3.3748    1.0000    4868.4     0.2258E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.055    3.3669    1.0000    4868.4     0.2172E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.864    3.3588    1.0000    4868.4     0.2082E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.676    3.3507    1.0000    4868.4     0.1989E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.490    3.3425    1.0000    4868.4     0.1894E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.307    3.3343    1.0000    4868.4     0.1796E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.126    3.3260    1.0000    4868.4     0.1697E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 75.000    3.7474    1.0000    4868.4     0.1814E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 70.000    3.7484    1.0000    4868.4     0.2885E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 65.000    3.7475    1.0000    4868.4     0.4202E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 60.000    3.7436    1.0000    4868.4     0.5639E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 55.000    3.7353    1.0000    4868.4     0.7244E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 50.000    3.7198    1.0000    4868.4     0.9846E-05   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 48.000    3.7107    1.0000    4868.4     0.1157E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 46.000    3.6994    1.0000    4868.4     0.1384E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 44.000    3.6855    1.0000    4868.4     0.1668E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 42.000    3.6684    1.0000    4868.4     0.1977E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.000    3.6473    1.0000    4868.4     0.2249E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 38.000    3.6213    1.0000    4868.4     0.2411E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.000    3.5893    1.0000    4868.4     0.2445E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.000    3.5497    1.0000    4868.4     0.2478E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.000    3.5007    1.0000    4868.4     0.2678E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.000    3.4400    1.0000    4868.4     0.2744E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.000    3.4043    1.0000    4868.4     0.2536E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.000    3.3645    1.0000    4868.4     0.2146E-04   -5.655000   68.361298   37.160000   79.059998 0  1 COMMENT
