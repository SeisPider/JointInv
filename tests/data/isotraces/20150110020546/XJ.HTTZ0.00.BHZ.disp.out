SURF96 R U T -1 75.000    3.7474    1.0000    
SURF96 R C T -1 75.000    6.8120    1.0000    
SURF96 R U T -1 70.000    3.7484    1.0000    
SURF96 R C T -1 70.000    6.4599    1.0000    
SURF96 R U T -1 65.000    3.7475    1.0000    
SURF96 R C T -1 65.000    6.1422    1.0000    
SURF96 R U T -1 60.000    3.7436    1.0000    
SURF96 R C T -1 60.000    5.8542    1.0000    
SURF96 R U T -1 55.000    3.7353    1.0000    
SURF96 R C T -1 55.000    5.5908    1.0000    
SURF96 R U T -1 50.000    3.7198    1.0000    
SURF96 R C T -1 50.000    5.3478    1.0000    
SURF96 R U T -1 48.000    3.7107    1.0000    
SURF96 R C T -1 48.000    5.2555    1.0000    
SURF96 R U T -1 46.000    3.6994    1.0000    
SURF96 R C T -1 46.000    5.1654    1.0000    
SURF96 R U T -1 44.000    3.6855    1.0000    
SURF96 R C T -1 44.000    5.0774    1.0000    
SURF96 R U T -1 42.000    3.6684    1.0000    
SURF96 R C T -1 42.000    4.9910    1.0000    
SURF96 R U T -1 40.000    3.6473    1.0000    
SURF96 R C T -1 40.000    4.9058    1.0000    
SURF96 R U T -1 38.000    3.6213    1.0000    
SURF96 R C T -1 38.000    4.8215    1.0000    
SURF96 R U T -1 36.000    3.5893    1.0000    
SURF96 R C T -1 36.000    4.7374    1.0000    
SURF96 R U T -1 34.000    3.5497    1.0000    
SURF96 R C T -1 34.000    4.6529    1.0000    
SURF96 R U T -1 32.000    3.5007    1.0000    
SURF96 R C T -1 32.000    4.5670    1.0000    
SURF96 R U T -1 30.000    3.4400    1.0000    
SURF96 R C T -1 30.000    4.4786    1.0000    
SURF96 R U T -1 29.000    3.4043    1.0000    
SURF96 R C T -1 29.000    4.4329    1.0000    
SURF96 R U T -1 28.000    3.3645    1.0000    
SURF96 R C T -1 28.000    4.3861    1.0000    
